# HG changeset patch
# User Betterbird <betterbird@betterbird.eu>
# Date 1664309152 -7200
# Parent  371bbd62177102fa8e4d2d5c97012b2e8b7191d4
Feature: Move startup folder string to Fluent.

diff --git a/mail/base/content/mainPopupSet.inc.xhtml b/mail/base/content/mainPopupSet.inc.xhtml
--- a/mail/base/content/mainPopupSet.inc.xhtml
+++ b/mail/base/content/mainPopupSet.inc.xhtml
@@ -571,17 +571,17 @@
     <menuitem id="folderPaneContext-favoriteFolder"
               type="checkbox"
               label="&folderContextFavoriteFolder.label;"
               accesskey="&folderContextFavoriteFolder.accesskey;"
               checked="false"
               oncommand="ToggleFavoriteFolderFlag();"/>
     <menuitem id="folderPaneContext-startupFolder"
               type="checkbox"
-              label="&folderContextStartupFolder.label;"
+              data-l10n-id="context-menu-startup-folder"
               checked="false"
               oncommand="ToggleStartupFolder();"/>
     <menuitem id="folderPaneContext-properties"
               label="&folderContextProperties2.label;"
               accesskey="&folderContextProperties2.accesskey;"
               oncommand="gFolderTreeController.editFolder();"/>
     <menuitem id="folderPaneContext-markAllFoldersRead"
               label="&folderContextMarkAllFoldersRead.label;"
diff --git a/mail/locales/en-US/chrome/messenger/messenger.dtd b/mail/locales/en-US/chrome/messenger/messenger.dtd
--- a/mail/locales/en-US/chrome/messenger/messenger.dtd
+++ b/mail/locales/en-US/chrome/messenger/messenger.dtd
@@ -647,17 +647,16 @@
 <!ENTITY folderContextSubscribe.label "Subscribe…">
 <!ENTITY folderContextSubscribe.accesskey "b">
 <!ENTITY folderContextSearchForMessages.label "Search Messages…">
 <!ENTITY folderContextSearchForMessages.accesskey "S">
 <!ENTITY folderContextProperties2.label "Properties">
 <!ENTITY folderContextProperties2.accesskey "P">
 <!ENTITY folderContextFavoriteFolder.label "Favorite Folder">
 <!ENTITY folderContextFavoriteFolder.accesskey "a">
-<!ENTITY folderContextStartupFolder.label "Startup Folder">
 <!ENTITY folderContextSettings2.label "Settings">
 <!ENTITY folderContextSettings2.accesskey "e">
 
 <!-- Search Bar -->
 <!ENTITY SearchNameOrEmail.label "Name or Email contains:">
 <!ENTITY SearchNameOrEmail.accesskey "N">
 
 <!-- Gloda Search Bar -->
diff --git a/mail/locales/en-US/messenger/messenger.ftl b/mail/locales/en-US/messenger/messenger.ftl
--- a/mail/locales/en-US/messenger/messenger.ftl
+++ b/mail/locales/en-US/messenger/messenger.ftl
@@ -130,16 +130,19 @@ mail-context-delete-messages =
      [one] Delete message
     *[other] Delete selected messages
   }
 
 context-menu-decrypt-to-folder =
     .label = Copy As Decrypted To
     .accesskey = y
 
+context-menu-startup-folder =
+    .label = Startup Folder
+
 ## Message header pane
 
 other-action-redirect-msg =
     .label = Redirect
 
 message-header-msg-flagged =
     .title = Starred
     .aria-label = Starred
